### DISCUSSION TITLE
Should we add a template for PRs?

### DISCUSSION THREAD

**Discussion by FabioRosado** [pyscript/pyscript#1083] [Proposals]
As I work on #1060 we will need to make sure that each PR has an entry to the changelog, this made me wonder if perhaps we should include a very basic template for new PRs. Something like:

```
Checklist:

- [ ] All tests pass locally
- [ ] I have updated `docs/changelog.md`
- [ ] I have created documentation for this(if applicable)
```

That way new folks will know what to do as soon as they open a PR.  Thoughts? 🤔

**Comment by JeffersGlass** [comment]
Yes please! This would be swell.
